<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatorLoginPage</class>
 <widget class="QWidget" name="CreatorLoginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Creator Login - VibeTube</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget {
    background-color: #FFE6F0;
    color: black; 
}
QLabel {
    font-size: 14px;
    font-weight: bold;
    color: black;
}
QLineEdit {
    background-color: #FFFFFF;
    border: 2px solid #FFC1CC;
    border-radius: 6px;
    padding: 4px;
}
QPushButton {
    background-color: #FF6F91;
    color: white;
    font-weight: bold;
    border-radius: 8px;
    padding: 6px 12px;
}
QPushButton:hover {
    background-color: #FF9AA2;
}
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="text">
      <string> Login</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="username_login">
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="email_login">
     <property name="placeholderText">
      <string>Email</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="pass_login">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="login_button">
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="back_button_login">
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
